build/buildBuildImages.sh

```diff
@@ -514,6 +556,8 @@ elif [ "$imageTypeToBuild" == "full" ]; then
 	buildFullImage "buster"
 elif [ "$imageTypeToBuild" == "vso-focal" ]; then
 	buildVsoFocalImage
```

Comment:
Looks like the `buildVsoImage` method you made can support a debian bullseye OS version or a ubuntu focal OS version. Should we remove the `buildVsoFocalImage` method and switch this to the `buildVsoImage` you made?

Reply:
Thank Paul, update has been made.